### ISSUE TITLE
Elixir example doesn't work

### ISSUE THREAD

**Issue by macifell** [g-andrade/tls_certificate_check#48]
This is pretty minor, but I noticed that the Elixir documentation doesn't actually work.
```
iex(107)> host = "example.com"
"example.com"
iex(108)> options = :tls_certificate_check.options(host)
[...]
iex(109)> :ssl.connect(host, 443, options, 5000)
{:error,
 {:options,
  {:socket_options,
   [packet_size: 0, packet: 0, header: 0, active: false, mode: :binary]}}}
```

Changing the double quotes to single quotes for the host value fixes the issue:
```
iex(110)> host = 'example.com'                          
'example.com'
iex(111)> options = :tls_certificate_check.options(host)
[...]
iex(112)> :ssl.connect(host, 443, options, 5000)        
{:ok,
 {:sslsocket, {:gen_tcp, #Port<0.342>, :tls_connection, :undefined},
  [#PID<0.4238.0>, #PID<0.4237.0>]}}
```

**Comment by g-andrade** [comment]
Thanks for pointing it out, it should work with double quotes - there must be a bug in converting the binary (Elixir strings) to a charlist (Erlang old strings).

**Comment by g-andrade** [comment]
Ohh, actually it's [`ssl:connect()`](https://www.erlang.org/doc/man/ssl#connect-4) that doesn't accept binary strings. That was silly of me. I'll fix the docs

**Comment by g-andrade** [comment]
Fixed in 6bba8d4. I kept the binary string and converted it to a charlist before passing it to `:ssl.connect/4`, as I expect that most integrations of `tls_certificate_check` in Elixir - other than working with `ssl` sockets directly - will idiomatically take binary strings.

**Comment by macifell** [comment]
Thank you @g-andrade!
